docs/source/workflows/show_sweeps.rst

```diff
@@ -78,19 +78,16 @@ Here is the output of the command:
    ┏━━━━━━━━━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━┳━━━━━━━━━━┳━━━━━━━━━━━━━━━┓
    ┃ name            ┃ status  ┃ framework         ┃ cloud_compute ┃ n_trials ┃ n_trials_done ┃
    ┡━━━━━━━━━━━━━━━━━╇━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━╇━━━━━━━━━━╇━━━━━━━━━━━━━━━┩
-   │ thomas-f5fee22a │ running │ pytorch_lightning │ cpu           │ 10       │ 5             │
+   │ thomas-5660535a │ succeeded │ pytorch_lightn… │ cpu-medium    │ 3        │ 3             │
```

Comment:
Looks like the lines in this table don't line up

Reply:
Updated. But sphinx handles it not perfectly. Not much I can do there.